patches/minim/0005-r14-Reset-indication-in-power-led.patch

```diff
@@ -0,0 +1,38 @@
+From 6ee0388954916442e2b92efd170db45a1243f3b7 Mon Sep 17 00:00:00 2001
+From: Sagar Jain <sagarj@minim.com>
+Date: Wed, 14 Dec 2022 22:57:46 +0530
+Subject: [PATCH 1/1] r14: Reset  indication in power led.
+
+SW-2878
+---
+ package/base-files/files/etc/rc.button/reset | 7 ++++---
+ 1 file changed, 4 insertions(+), 3 deletions(-)
+
+diff --git a/package/base-files/files/etc/rc.button/reset b/package/base-files/files/etc/rc.button/reset
+index 2403122ad2..43ffae0a18 100755
+--- a/package/base-files/files/etc/rc.button/reset
++++ b/package/base-files/files/etc/rc.button/reset
+@@ -7,8 +7,9 @@ OVERLAY="$( grep ' /overlay ' /proc/mounts )"
+ case "$ACTION" in
+ pressed)
+ 	[ -z "$OVERLAY" ] && return 0
+-
+-	return 5
++	sleep 10
```

Comment:
I'm not sure about this first sleep. If a user presses the button, but releases it before the 10 seconds is up, then we'd get stuck in here and start flashing the LEDs after the sleep ends, even though the user didn't hold long enough for factory reset condition.

Reply:
That sleep is unreachable.

Should just return 10 and the architecture will take care of the timing.

Reply:
@sagarjminim try to find some doco on these scripts, you're doing the work that the infrastructure does for you.  Take a look at the 7600 and do the same here.

Reply:
Working on it.